source/mainnet/net/resources/release-notes-mainnet.rst

```diff
@@ -11,6 +11,13 @@ Release notes - Mainnet
 
    To subscribe to updates on the Mainnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.
 
+April 21, 2023
+
+Cargo-concordium integration testing
+-------------------------------------
+
+Important new functionality has been added to ``cargo-concordium``: the XX library makes it possible to create and run automatic integration tests of smart contracts. This will allow a smart contract developer to write code that runs multiple contracts in a locally-controlled environment, interacts with them, and asserts that the eventual output and state of the contracts are as expected. For more information about how to enable this, see :ref:`Integration test a contract in Rust<integration-test-contract>`.
```

Comment:
We should phrase this differently. It is not really part of cargo-concordium. It is a separate library.

So I would recommend adding a header "smart contract integration testing" and phrase this in terms of that.

cargo-concordium is largely orthogonal to this.

Reply:
Yes, it is on the todo list for today and listed in the PR description ;)
![image](https://user-images.githubusercontent.com/5469652/233598878-15b9f39a-c796-48bc-81ca-42389249462a.png)

Reply:
I just pushed the commit with this change